eq-author/src/App/sharing/SharePageContent/EditorSearch.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { flowRight } from "lodash";
+
+import { Query, useMutation } from "react-apollo";
+
+import ALL_USERS from "../graphql/AllUsers.graphql";
+import ADD_REMOVE_EDITOR from "../graphql/AddRemoveEditor.graphql";
+
+import Loading from "components/Loading";
+import Error from "components/Error";
+import UserList from "./UserList";
+import UserSearch from "./UserSearch";
+
+import { Section, EditorTitle, Described } from "../styles/SharePageContent";
+
+const User = PropTypes.shape({
+  picture: PropTypes.string,
+  name: PropTypes.string,
+  email: PropTypes.string,
+  id: PropTypes.string,
+});
+
+const propType = {
+  EditorSearch: {
+    questionnaireId: PropTypes.string,
+    users: PropTypes.arrayOf(User),
+    owner: PropTypes.shape({
+      picture: PropTypes.string,
+      name: PropTypes.string,
+      email: PropTypes.string,
+      id: PropTypes.string,
+    }),
+    editors: PropTypes.arrayOf(User),
+    showToast: PropTypes.func.isRequired,
+  },
+  GetUserWrapper: {
+    questionnaireId: PropTypes.string,
+    users: PropTypes.arrayOf(User),
+    owner: PropTypes.shape({
+      picture: PropTypes.string,
+      name: PropTypes.string,
+      email: PropTypes.string,
+      id: PropTypes.string,
+    }),
+    editors: PropTypes.arrayOf(PropTypes.shape(User)),
+  },
+};
```

Comment:
The `propType` object is all components, would putting the User one in there confuse everything? It's only been taken out because it's reused so much.

Reply:
Ahh okay. No, that makes sense. Fair shout mate.